modules/core/src/test/java/org/apache/ignite/internal/processors/odbc/SqlListenerUtilsTest.java

```diff
@@ -48,6 +48,9 @@ public void translateWildchar() {
         check(".ome", "_ome");
 
         check("so_e", "so\\_e");
+        check("so__e", "so\\_\\_e");
+        check("so__e___f", "so\\_\\_e\\_\\_\\_f");
```

Comment:
Let's also check other series works well
some\\\\_\\\\_table -> some\\.\\.table
some\\\\\\_\\\\\\_table -> some\\_\\_table

Reply:
@AMashenkov  I updated the impl, please help to review, thanks. The things need to discuss is that the test cases about the `translateSeriesOfBackslashes`, that would fail currently